monero-outreach-docs/translations/nl/quickfacts/quickfacts.md

```diff
@@ -0,0 +1,135 @@
+
+_Met een staartemissie van 0.6 XMR/blok zal er in 2040 evenveel Monero als Bitcoin zijn (ongeveer 21 miljoen)._
+
+
+### TOEKOMSTIGE FEATURES
+
+Hoewel Monero beschikbaar is en al gebruikt wordt over de hele wereld, hebben de programmeurs interessante plannen om privacy, veiligheid, en functionaliteit van Monero (en cryptovaluta in het algemeen) te blijven verbeteren. Onderstaande features komen er binnenkort aan:
+
+**Bulletproofs:** Deze [technologie](https://getmonero.org/es/2017/12/07/Monero-Compatible-Bulletproofs.html) slaat de blokketen efficiënter op en biedt snellere verifiëring van transacties.
+
+**Kovri:** Voor privacy is dit een belangrijke upgrade. Kovri voegt een vierde laag privacy toe aan Monero-transacties. Hiertoe gebruikt het **[garlic-encryption](https://getmonero.org/resources/moneropedia/garlic-encryption.html)** (i.e. TOR's grote broer) door het IP-adres van zenders en ontvangers te verbergen. Elke laag van een Monero-transactie zal privé en veilig zijn: van zender, ontvanger, bedrag, tot het überhaupt gebruik van het Moneronetwerk (dankzij Kovri).
+
+**Hardware Wallets:** Trezor Model T hardware wallet **[ondersteunt gebruik van Monero](https://trezor.io/coins/#XMR)**. **[Ledger hardware wallet](https://support.ledgerwallet.com/hc/en-us/articles/360006352934-Monero-XMR-Advanced)** is al geïntegreerd via de officiële Monero GUI wallet. De gemeenschap sponsert een team dat **[Kastelo, een open-bron hardware wallet](http://kastelo.org/)** bouwt, volgens de Monero-ethos. Beide opties worden volledig ondersteund.
+
+**Kasisto:** Dit **[computergestuurd kasregister](https://github.com/amiuhle/kasisto)** is ontworpen voor verkoopsgemak van handelaars en 
+bedrijven.
+
+## EXTRA BRONMATERIAAL
+
++ **_[getmonero.org (Officiële website)](https://getmonero.org/)_**
++ **_[monero.how](https://www.monero.how/)_**
++ **_[reddit.com/r/monero](https://www.reddit.com/r/Monero/)_**
++ **_[Gids voor Monero (post)](https://www.reddit.com/r/CryptoCurrency/comments/7ra409/your_guide_to_monero_and_why_it_has_great/)_**
++ **_[diepgaande technische introductie tot Monero](https://steemit.com/monero/@sgp/7yjqso-a-monero-introduction-for-beginners)_**
++ **_[Zero to Monero](https://www.getmonero.org/library/Zero-to-Monero-1-0-0.pdf)_**
+
++ **_[Vermijd deze oplichters](https://www.reddit.com/r/Monero/wiki/avoid)_**
++ **_[Monero FAQ](https://ww.getmonero.org/get-started/faq/)_**
++ **_[Monero SWOT analyse]()_**
++ **_[de Monero gemeenschap](https://getmonero.org/community/hangouts/)_**
++ **_[Mastering Monero](https://masteringmonero.com/)_**
+
+**Monero Quick Facts - Gereviseerd op 14/11/2018**    
+**Geschreven voor de Monero-gemeenschap door Monero Outreach.**
+
+**Monero Outreach heeft als doel om adoptie en gebruik van Monero te stimuleren door middel van educatie en public relations. Jouw donaties maken dit mogelijk.**
```

Comment:
This paragraph and the previous one are the same in @MrGmv's translation, so he should be credited.